frontend/src/constants.js

```diff
@@ -5,6 +5,7 @@ export const CODALAB_VERSION = '0.5.26';
 export const NAME_REGEX = /^[a-zA-Z_][a-zA-Z0-9_.-]*$/i;
 
 export const NAVBAR_HEIGHT = 60;
+export const HEADER_HEIGHT = 170;
```

Comment:
So I guess the header height is hard coded 170 for the header component? Make sure to update there to use this constant as well so things don't go out of sync